lib/tests-from-pull-request.js

```diff
@@ -28,28 +28,35 @@ function getTestsFromPR(context) {
   }
 }
 
-function getTestNames(prPatch) {
+function getTestNames(prPatch, context) {
   const testNames = [];
   const parsedPatch = prPatch.split('\n');
-
-  parsedPatch.forEach((line) => {
-    if (line.startsWith('+')) {
-      line = removePrefixAndTrim(line);
-      const testName = getTestName(line);
-
-      testName ? testNames.push(testName) : null;
-    }
-  });
-
-  return testNames;
+  return getConfig(context)
```

Comment:
I think it might be better to pass this as the parameter for the `getTestNames` function instead of passing the context and extracting it. It will make it so that `getTestsFromPR` knows the context, but `getTestNames` doesn't even know the context exists and just runs on parameters. It will make tests a lot easier to write :-)